web-ng/src/app/components/layer-dialog/layer-dialog.component.html

```diff
@@ -96,6 +96,11 @@
               Add option
             </a>
           </div>
+          <div fxLayout fxLayoutAlign="end" class="question-actions">
+            <mat-icon (click)="deleteQuestion(j)" class="delete-icon">
```

Comment:
Please use `mat-icon-button` to get appropriate Material hover and click behavior:
https://material.angular.io/components/button/overview